src/GraphQlClientGenerator/BaseClasses.cs

```diff
@@ -841,7 +845,7 @@ public GraphQlFragmentCriteria(GraphQlQueryBuilder objectQueryBuilder, GraphQlDi
     }
 }
 
-public abstract class GraphQlQueryBuilder<TQueryBuilder> : GraphQlQueryBuilder where TQueryBuilder : GraphQlQueryBuilder<TQueryBuilder>
+public abstract partial class GraphQlQueryBuilder<TQueryBuilder> : GraphQlQueryBuilder where TQueryBuilder : GraphQlQueryBuilder<TQueryBuilder>
```

Comment:
Instead of adding `WithGenericFields` method into the base class, I made the base class `partial` and added the missing method in our code base. Will change all base classes to `partial` to allow furthere extensibility